### ISSUE TITLE
[Bug]: Timeout to low

### ISSUE THREAD

**Issue by sinnbeck** [pestphp/pest#1233]
### What Happened

I am working with a filament app, but apparently the default timeout is too low. I am running in parallel, but almost all tests are timing out which means I cannot trust my tests at all

![image](https://github.com/user-attachments/assets/1fce49c9-1def-4d82-93b4-d8f635fbbda2)

### How to Reproduce

Have a complex filament page with a lot of form fields

### Pest Version

3.0.4

### PHP Version

8.3

### Operation System

Linux

### Notes

I am running the tests in docker (Lando).

It might be practical to allow overwriting the timeout with a flag or a setting in Pest.php

If I just manually change the timeout in the vendor folder it works
![image](https://github.com/user-attachments/assets/503e0275-9502-4326-b837-c3815084c4c9)

**Comment by danielh-official** [comment]
Could this work for you? `--default-time-limit <sec>`

https://pestphp.com/docs/cli-api-reference#content-execution

**Comment by sinnbeck** [comment]
@danielh-official thanks for the suggestion but I just get `The "--default-time-limit" option does not exist.`

And I doubt it would work, as the values are hardcoded currently.
https://github.com/pestphp/pest-plugin-mutate/blob/3.x/src/MutationTest.php#L108

**Comment by gehrisandro** [comment]
Hi @sinnbeck 

Thanks for raising the issue.

Looks like we are missing the correct implementation there. I am going to fix that.

**Comment by gehrisandro** [comment]
Issue will be fixed in the next release 
https://github.com/pestphp/pest-plugin-mutate/pull/15

**Comment by gehrisandro** [comment]
@nunomaduro This issue can be closed.

**Comment by jonathanpmartins** [comment]
@gehrisandro  Is there a way that we could set a custom timeout? Or a custom timeout offset on top of the calculated one?

**Comment by sinnbeck** [comment]
I will see how the new version works and make a new issue if I run into any issues with that :)
